horovod/spark/torch/remote.py

```diff
@@ -266,9 +267,16 @@ def transform_outputs(outputs, labels):
 
                         # reshape labels to match the output shape of the model
                         if hasattr(outputs[0], 'shape'):
-                            labels = [label.reshape(output.shape)
-                                      if output.shape.numel() == label.shape.numel() else label
-                                      for label, output in zip(labels, outputs)]
+                            # If label_shapes parameter is not provided, reshape the label columns
```

Comment:
Can move this comment into the `else` branch so it's closer to the code it's describing.